docs/detections/session-view.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[session-view]]
+== Session view
+Session View, currently in beta, is an investigation tool that allows you to view and analyze Linux process data organized
+in a tree according to the Linux logical event model, with processes organized by parentage and time of execution.
+It displays events in a highly readable format inspired by the terminal, making it a powerful tool for monitoring
+and investigating session activity on your Linux infrastructure, and understanding user and service behaviour.
+
+[float]
+[[session-view-data]]
+====== Session View displays:
+1. *Interactive and non-interactive processes:* processes and services with or without a controlling tty.
+2. *User information:* the Linux user that executed each session or process, and any exec user changes.
+3. *Process and event telemetry:* process information included in the Linux logical event model.
+4. *Nested sessions:* sessions started by processes descended from the entry session.
+5. *Alerts:* alerts in the context of the processes which caused them.
+
+[float]
+[[enable-session-view]]
+=== Enable session view data
+Session View uses process data collected by the {endpoint-sec} integration,
+but this data is not collected by default. To enable Session View data, navigate to the Endpoint Security
+integration and edit one or more of your integration policies. Search the policy's settings for “session data”,
+and enable “Include session data” near the bottom of the page. Only data collected by {endpoint-sec} with this setting
+enabled can be viewed in Session View.  For more information about the additional
+fields collected by the {endpoint-sec} when this setting is enabled, see https://www.elastic.co/[placeholder link].
+
+[float]
+[[open-session-view]]
+=== Open session view
+Session View is accessible from the Hosts, Alerts, and Timelines pages.
+In general, to open Session View you will first find an event or session you wish to view,
+then click *Open Session View* under *Actions*. For example:
+
+[float]
+[[find-sv-on-alerts-pg]]
+===== On the Alerts page
+* Go to *Detect* -> *Alerts*, then scroll down to view the Alerts table.
+Events viewable in Session View have a rectangular **Open Session View** icon under **Actions**:
+[role="screenshot"]
+image::images/session-view-action-icon-detail.png[]
```

Comment:
This image looks really big in the preview. Maybe adjust display -- i think this is how? Also need to add alt text for all images:
```suggestion
image::images/session-view-action-icon-detail.png[Detail of Open Session View icon,width=75%]
```